config/install/image.style.logo.yml

```diff
@@ -9,6 +9,6 @@ effects:
     id: image_scale
     weight: 1
     data:
-      width: 120
-      height: 46
+      width: null
+      height: 92
```

Comment:
It's Max-height 46px in css, set 92 to make it look good on retina displays (blurry otherwise)